### DISCUSSION TITLE
does fastlane support video app preview update?

### DISCUSSION THREAD

**Discussion by schunlee** [fastlane/fastlane#17874] [Q&A] [answered]
Does fastlane support video preview upload? If I put an mp4 file in fastlane/screenshots directory, I find my video not be uploaded. If Fastlane deliver not support this, how about spaceship, thanks.

**Comment by ainame** [comment, marked as answer]
As far as I'm aware, `deliver` doesn't support "App Previews" video to upload yet but `spaceship` does have `Spaceship::ConnectAPI::AppPreview` and `Spaceship::ConnectAPI::AppPreviewSet` models. You might be able to use it for your purpose?

**Reply by schunlee** [reply]
thank you very much.
